data/fixtures/recorded/languages/markdown/roundWrapItemAir.yml

```diff
@@ -26,8 +26,8 @@ initialState:
       end: {line: 0, character: 5}
 finalState:
   documentContents: |
-    - (aaa)
-      - bbb
+    - (aaa
```

Comment:
You now need to use `line item` to get the old behavior

Reply:
Probably better to include the children by default (the new way) because you can easily narrow the scope but it's harder to widen it arbitrarily